psgdpr.php

```diff
@@ -537,6 +537,10 @@ public function loadRegisteredModules()
         $module_list = array();
         foreach ($modules as $module) {
             $Module = Module::getInstanceById($module['id_module']);
+            
+            if ($Module == false) {
```

Comment:
```suggestion
            if ($moduleInstance === false) {
```
Can you rename `$Module` into `$moduleInstance` please

Reply:
Done